source/index.html

```diff
@@ -4,9 +4,169 @@
     <meta charset="utf-8">
     <title>HTML Academy: {{projectTitle}}</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
-  </body>
+  <body class="page-body">
+    <header class="main-header">
+      <a href="#" class="main-header__logo">
+        <img class="main-header__logo-img" src="" alt="Логотип Кэт энерджи." width: 33.32px; height: 37.61px; >
+        <img src="" alt="Логотип Кэт энерджи." class="main-header__logo-img" >
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню.</span>
+        </button>
+      </nav>
+      <div class="main-nav__wrapper">
+        <ul class="site-list">
+<li class="site-list__item">
+  <a href="#" class="site-list__item-link">
+    Главная
+  </a>
+</li>
+<li class="site-list__item">
+  <a href="#" class="site-list__item-link">
+Каталог продукции
+  </a>
+</li>
+<li class="site-list__item">
+  <a href="#" class="site-list__item-link">
+  Подбор программы
+  </a>
+</li>
+  </ul>
+      </div>
+    </header>
+<main class="main-container">
+  <section class="product"> 
+  <h1 class="product__title">
+    Функциональное питание для котов
+  </h1>
+  <p class="product__subtitle">
+    Занялся собой? Займись котом!
+  </p>
+  <img src="#" alt="Кэт Энерджи с курицей." class="product__image">
+  <button class="product__button button">
+    Подобрать программу
+  </button>
+</section>
+<section class="main-product-list">
+  <h2 class="visually-hidden">
+    Варианты программ.
+  </h2>
+  <ul class="product-list">
+    <li class="product-list__item product-list__item--slim">
+      <h2 class="product-list__title">
+        Похудение
+      </h2>
+      <p class="product-list__subtitle">
+        Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора на диету! Cat Energy Slim поможет вашему питомцу сбросить лишний вес.
+      </p>
+<a href="#" class="product-list__link product-list__link-arrow">
```

Comment:
```suggestion
<a href="#" class="product-list__link">
```
зачем тут это? типа чтоб стрелку на этот класс повесить? так повесь ее на link просто и все